ONSdigital/dp-frontend-router
# intercept dataset landing pages

### What

- Add the ability to intercept legacy and new (filterable) dataset landing pages

### How to review

- Checkout, build and run all of the following services: dp-frontend-router, dp-frontend-dataset-controller, dp-frontend-filter-dataset-controller, dp-frontent-renderer all on branch feature/intercept-datasets
- Ensure you have sixteens, zebedee and babbage all running
- Ensure you have go-ns checked out on feature/zebedee client
- navigate your browser to http://localhost:20000/economy/economicoutputandproductivity/productivitymeasures/datasets/labourproductivitybyindustrydivision to test the legacy dataset landing page
- navigate your browser to http://localhost:20000/datasets/1234/editions/5678/versions/2017 to test the new dataset landing page (note this is a completely stubbed page with no frontend model as yet)
- Related reviews:

https://github.com/ONSdigital/dp-frontend-renderer/pull/55
https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/pull/6
https://github.com/ONSdigital/dp-frontend-dataset-controller/pull/1
https://github.com/ONSdigital/dp-frontend-router/pull/26
https://github.com/ONSdigital/go-ns/pull/11

### Who can review

Anyone
